distributor.sample.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<composer-distributor xmlns="https://phar.io/composer-distributor"
+                      packageName="phar-io/phive"
+                      keyDirectory="keys">
+
+  <phar name="phive"
```

Comment:
Do we actually need the `name` attribute?

Reply:
Yes for the name if the binary since one package can hold multiple PHAR files